MiniMonsterGame/MonsterGame/Assets/Scripts/IClick.cs

```diff
@@ -0,0 +1,8 @@
+﻿using UnityEngine;
+
+public delegate void ClcikEventHandler(GameObject selected);
+public interface IClick
+{  
+    event ClcikEventHandler ClickEvent;
+}
```

Comment:
```suggestion

using System;

public interface IClick
{  
    public event Action<GameObject> ClickEvent;
}
```

이런건 어떨까요? 다만 IClick이 한곳에만 쓰이네요. 인터페이스를 써야 할 이유가 약한것 같습니다.

Reply:
맞습니다! 프로젝트 초기에는 인터페이스와 이벤트를 활용해보겠어!가 단순 목표였기 때문입니다. 이제 더 적절한 곳에 활용될 수 있도록 방안을 찾아보겠습니다.